SonarQube.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -302,9 +302,10 @@
        any values passed on the MSBuild command line or in <PropertyGroups>.
        -->
 
-  <!-- The FxCop targets are conditionally imported so we need to make sure the required condition
-       is true early enough. -->
-  <PropertyGroup Condition=" $(SonarQubeTempPath) != '' ">
+  <!-- The FxCop targets are conditionally imported so we need to make sure the required condition is true early enough. -->
+  <!-- SONARMSBRU-193 On C++ projects setting RunCodeAnalysisOnce has unwanted side-effects -->
+  <!-- Note: $(Language) is already available here. For order evaluation details, see: msbuild /preprocess:out.txt project.proj -->
+  <PropertyGroup Condition=" $(SonarQubeTempPath) != '' AND ( $(Language) == 'C#' OR $(Language) == 'VB' ) ">
```

Comment:
this real fix is here: `RunCodeAnalysisOnce` is now only set when `$(Language)` is either `C#` or `VB` to avoid any unwanted side-effect